accounts/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login, authenticate
+from django.contrib.auth.forms import UserCreationForm
+
+
```

Comment:
What do you think about adding this ?
```suggestion
from django.views.decorators.http import require_http_methods

@require_http_methods(["GET", "POST"])
```